src/components/Link.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { routeTo } from 'redux-router-kit';
+
+const LinkView = ({ targetTo, text, path }) =>
+  (<a
+    tabIndex="0"
+    role="link"
+    onClick={(event) => {
+      event.preventDefault();
+      targetTo(path);
+    }}
+  >{text}</a>);
```

Comment:
Using the [`children`](https://facebook.github.io/react/docs/composition-vs-inheritance.html) property instead of `text` will make the component even more useful.